server/connectors/gcp_connector/gcp_post_auth_tasks.py

```
@@ -78,34 +71,9 @@ def gcp_post_auth_setup_task(self, user_id, selected_project_ids=None):
-        # Determine which projects to setup based on selection
-        if selected_project_ids:
-            # Filter projects to only the selected ones
-            projects_to_setup = [p for p in projects if p.get('projectId') in selected_project_ids]
-            logging.info(f"Running setup for {len(projects_to_setup)} selected projects")
-        else:
-            # Use all projects
-            projects_to_setup = projects
-            logging.info(f"Running setup for all {len(projects_to_setup)} projects")
+        # Filter to selected projects
+        projects_to_setup = [p for p in projects if p.get('projectId') in selected_project_ids]
+        logging.info(f"Running setup for {len(projects_to_setup)} selected projects")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)